src/Yoti/Util/Profile/AttributeConverter.php

```diff
@@ -8,13 +8,15 @@
 use Compubapi\EncryptedData;
 use Attrpubapi\Attribute as ProtobufAttribute;
 use Yoti\Exception\AttributeException;
+use Yoti\Entity\MultiValue;
 
 class AttributeConverter
 {
     const CONTENT_TYPE_JSON = 5;
     const CONTENT_TYPE_PNG = 4;
     const CONTENT_TYPE_JPEG = 2;
     const CONTENT_TYPE_DATE = 3;
+    const CONTENT_TYPE_MULTI_VALUE = 6;
```

Comment:
I thought these could be alphabetical, but doesn't seem like there's any order. Could we change them to order by the enum value?

Reply:
Fixed in 152f09b